app/views/helpers/extension/details.phtml

```diff
@@ -1,22 +1,16 @@
 <?php /** @var FreshRSS_View $this */ ?>
-<ul class="horizontal-list">
-	<li class="item">
-		<?php if ($this->ext_details->getType() === 'user' || FreshRSS_Auth::hasAccess('admin')) { ?>
-		<?php $name_encoded = urlencode($this->ext_details->getName()); ?>
-		<div class="stick">
-			<a class="btn open-slider" href="<?= _url('extension', 'configure', 'e', $name_encoded) ?>"><?= _i('configure') ?> <?= _t('gen.action.manage') ?></a>
-			<?php if ($this->ext_details->isEnabled()) { ?>
-			<button class="btn active" form="form-extension" formaction="<?= _url('extension', 'disable', 'e', $name_encoded) ?>"><?= _t('gen.action.disable') ?></button>
-			<?php } else { ?>
-			<button class="btn" form="form-extension" formaction="<?= _url('extension', 'enable', 'e', $name_encoded) ?>"><?= _t('gen.action.enable') ?></button>
-			<?php } ?>
-			<?php if (FreshRSS_Auth::hasAccess('admin')) { ?>
-			<button class="btn btn-attention confirm" form="form-extension" formaction="<?= _url('extension', 'remove', 'e', $name_encoded) ?>"><?= _t('gen.action.remove') ?></button>
-			<?php } ?>
-		</div>
-		<?php } else { ?>
-		<?= _t('admin.extensions.system.no_rights') ?>
-		<?php } ?>
-	</li>
-	<li class="item"><?= $this->ext_details->getName() ?></li>
-</ul>
+
+<?php 
+if ($this->ext_details->getType() === 'user' || FreshRSS_Auth::hasAccess('admin')) { 
+	$name_encoded = urlencode($this->ext_details->getName()); 
+	$ext_enabled = $this->ext_details->isEnabled();
+	if ($ext_enabled) { ?>
+	<button class="switch active" form="form-extension" formaction="<?= _url('extension', 'disable', 'e', $name_encoded) ?>" title="<?= _t('gen.action.disable') ?>"></button>
+	<?php } else { ?>
+	<button class="switch" form="form-extension" formaction="<?= _url('extension', 'enable', 'e', $name_encoded) ?>" title="<?= _t('gen.action.enable') ?>"></button>
+	<?php } ?>
+<?php } else { ?>
+<?= _t('admin.extensions.system.no_rights') ?>
+<?php } ?>
+<a class="open-slider" title="<?= _t('gen.action.manage') ?>" href="<?= _url('extension', 'configure', 'e', $name_encoded) ?>"><?= _i('configure') ?></a>
```

Comment:
Note that `_i('configure')` does not at this point provide the context for screen readers that the text "Manage" does, only the less helpful text "gear" (as `⚙`).

Reply:
PS That can be split off into a separate ticket; it's not directly related.

Reply:
the link has now an `title`.

If we want to improve the config icons than we should improve it everywhere. Please check the other cog icons on other places.

Reply:
Icons, not cog icons.

Reply:
I do not understand what you mean.
`_i('configure')` is a cog icon. The wrapped `<a>` has an `title="manage"`.

Reply:
It's not important. You said other cog icons, but the problem (if it's a problem) will exist on any type of icon. ^_^

Off the cuff I think it should probably be something closer to `_i('configure', 'Manage')` (where `Manage` would be the alt and title text for the image, no worrying about the surrounding link/button necessary).